### PR TITLE
feat[w2.1]:add controller,html,js and css files about upload image

### PR DESCRIPTION
**Pull request by sladjana-savkovic**
- I have a goal to upload patient image in memory.
- At first, I need to make html, js, css and controller files.
- Afterwards, I must add new field in Patient class to remember a image name.

### CODE REVIEW

**Comment by vlaksi** [comment]
Great description for the pull request!

File: HealthcareSystem/PatientWebApp/Controllers/FileController.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using PatientWebApp.DTOs;
+
+namespace PatientWebApp.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class FileController : ControllerBase
```
Comment (vlaksi) [inline HealthcareSystem/PatientWebApp/Controllers/FileController.cs]: What **FileController** should do exactly?

Comment (sladjana-savkovic) [reply HealthcareSystem/PatientWebApp/Controllers/FileController.cs]: I'm currently refactoring code, so it will be different in the next commit.

**Comment by sladjana-savkovic** [comment]
[(8) W2.1 As a patient I want to register on the system so that I could use the services of the hospital.](https://trello.com/c/gBBlgnJG/21-8-w21-as-a-patient-i-want-to-register-on-the-system-so-that-i-could-use-the-services-of-the-hospital)

File: HealthcareSystem/PatientWebApp/Controllers/FileController.cs (hunk 1 above)
Comment (vlaksi) [reply HealthcareSystem/PatientWebApp/Controllers/FileController.cs]: Okay, feel free to inform me, when you are done. So I can look at the code.

**Review by sladjana-savkovic — COMMENTED** [review]
At this point, I think I'm done all about this pull request.

**Review by vlaksi — APPROVED** [review]
Everything looks fine to me.

**Review by jelenabudisa — APPROVED** [review]

**Pull request merged**